jxline-core/src/main/java/cloud/xline/jxline/impl/ClientConnectionManager.java

```diff
@@ -144,6 +149,37 @@ public void start(
         return channelBuilder;
     }
 
+    public ManagedChannel getInitChannel() {
+        if (this.initChannel == null) {
+            synchronized (lock) {
+                if (this.initChannel == null) {
+                    this.initChannel = defaultChannelBuilder().build();
+                }
+            }
+        }
+        return this.initChannel;
+    }
+
+    /**
+     * Create stub with saved integrated channel. Used to create some xline servers
```

Comment:
"Create some xline servers"? I don't know the meaning of this statement and can't guess it as `newStub` is not used in this PR.

Reply:
Some clients can only connect to the xline server, not the curp server. This type of client is not included in the current PR, so it is not used. It is used in the watch client, maintenace client and cluster client. They can use not all endpoints, just need to connect one of them, instead of like curp client needs to connect all endpoints.